ui/src/data-services/models/job.ts

```diff
@@ -4,56 +4,102 @@ import { STRING, translate } from 'utils/language'
 
 export type ServerJob = any // TODO: Update this type
 
+export enum JobStatus {
+  Pending = 'pending',
+  Started = 'started',
+  Success = 'success',
+  Unknown = 'unknown',
+}
+
 export class Job {
-  private readonly _job: ServerJob
+  protected readonly _job: ServerJob
 
   public constructor(job: ServerJob) {
     this._job = job
   }
 
-  get description(): string {
-    return this._job.description
+  get finishedAt(): string | undefined {
+    if (!this._job.finished_at) {
+      return
+    }
+
+    const date = new Date(this._job.finished_at)
+    const dateString = getFormatedDateString({ date })
+    const timeString = getFormatedTimeString({ date })
+
+    return `${dateString} ${timeString}`
   }
 
   get id(): string {
-    return this._job.id
+    return `${this._job.id}`
   }
 
-  get idLabel(): string {
-    return `#${this.id}`
-  }
+  get startedAt(): string | undefined {
+    if (!this._job.started_at) {
+      return
+    }
 
-  get jobStarted(): string {
-    const date = new Date(this._job.job_started)
+    const date = new Date(this._job.started_at)
     const dateString = getFormatedDateString({ date })
     const timeString = getFormatedTimeString({ date })
 
     return `${dateString} ${timeString}`
   }
 
-  get project(): string {
-    return this._job.project
+  get name(): string {
+    return this._job.name
   }
 
-  get totalImages(): number {
-    return this._job.total_images
+  get project(): string {
+    return this._job.project.name
   }
 
-  get status(): number {
-    return this._job.status
+  get status(): JobStatus {
+    return this.getStatus(this._job.status)
   }
 
   get statusDetails(): string {
-    return 'More details about the job status.'
+    // TODO: Return BE value here when available.
+    return `${this.statusValue * 100}% completed.`
+  }
+
+  get statusValue(): number {
+    // TODO: Return BE value here when available (number between 0 and 1).
+    switch (this.status) {
```

Comment:
Mocked a value based on status for now. This value will control status bar size:

<img width="706" alt="Screenshot 2023-07-07 at 12 50 26" src="https://github.com/RolnickLab/ami-platform/assets/11680517/da58130f-8528-4984-9fba-95489e9d924e">

Reply:
I added a `summary` section to the `progress` JSON with the overall progress & status

<img width="376" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/158175/4b3d1619-c38c-44f0-94e1-262ce4093c84">